src/main.rs

```diff
@@ -19,12 +19,12 @@ async fn main() -> Result<(), GatewayError> {
     }
 
     // setup logging
-    println!("Initializing logging facilities");
+    log::info!("Initializing logging facilities");
```

Comment:
Te makra mają jakiś fallback zanim zainicjalizujemy logger?

Reply:
Tę linijkę to chyba można wywalić, bo raczej nikt tego nie zobaczy kiedy uruchomi jako daemon.